parquet-common/src/main/java/org/apache/parquet/Log.java

```diff
@@ -41,6 +34,7 @@
  * @author Julien Le Dem
  *
  */
+@Deprecated
```

Comment:
No problem. I'll open a follow-up issue for further discussion. We can deprecate and remove the use of this class for that issue instead of dragging out this one.